### ISSUE TITLE
Scryber.PDFRenderException: Method not found

### ISSUE THREAD

**Issue by creativesuspects** [richard-scryber/scryber.core#86]
**Describe the bug**
My project uses SixLabors.ImageSharp.Web 2.0.2 to pre-process images. When using Scryber.Core.Mvc 6.0.0-beta8 I get the following error whenever the HTML template contains an `<img />`, even when the image is served from an external CDN and not processed by ImageSharp:

```
Scryber.PDFRenderException: Could not output the block 'div :div 5:Block0'. Method not found: 'System.Span`1<!0> SixLabors.ImageSharp.Image`1.GetPixelRowSpan(Int32)'.
 ---> System.MissingMethodException: Method not found: 'System.Span`1<!0> SixLabors.ImageSharp.Image`1.GetPixelRowSpan(Int32)'.
   at Scryber.Imaging.Formatted.PDFImageSharpRGBA32Data.DoRenderImageData(IStreamFilter[] filters, ContextBase context, PDFWriter writer)
   at Scryber.Imaging.PDFImageDataBase.Render(PDFName name, IStreamFilter[] filters, ContextBase context, PDFWriter writer)
   at Scryber.PDF.Resources.PDFImageXObject.DoRenderToPDF(ContextBase context, PDFWriter writer)
   at Scryber.PDF.Resources.PDFResource.EnsureRendered(ContextBase context, PDFWriter writer)
   at Scryber.Components.ImageBase.OutputToPDF(PDFRenderContext context, PDFWriter writer)
   at Scryber.PDF.Layout.PDFLayoutComponentRun.DoOutputToPDF(PDFRenderContext context, PDFWriter writer)
   at Scryber.PDF.Layout.PDFLayoutLine.DoOutputToPDF(PDFRenderContext context, PDFWriter writer)
   at Scryber.PDF.Layout.PDFLayoutRegion.DoOutputToPDF(PDFRenderContext context, PDFWriter writer)
   at Scryber.PDF.Layout.PDFLayoutBlock.OutputInnerContent(PDFRenderContext context, PDFWriter writer)
   at Scryber.PDF.Layout.PDFLayoutBlock.DoOutputToPDF(PDFRenderContext context, PDFWriter writer)
   --- End of inner exception stack trace ---
   at Scryber.PDF.Layout.PDFLayoutBlock.DoOutputToPDF(PDFRenderContext context, PDFWriter writer)
   at Scryber.PDF.Layout.PDFLayoutRegion.DoOutputToPDF(PDFRenderContext context, PDFWriter writer)
   at Scryber.PDF.Layout.PDFLayoutPositionedRegionRun.DoOutputToPDF(PDFRenderContext context, PDFWriter writer)
   at Scryber.PDF.Layout.PDFLayoutLine.DoOutputToPDF(PDFRenderContext context, PDFWriter writer)
   at Scryber.PDF.Layout.PDFLayoutRegion.DoOutputToPDF(PDFRenderContext context, PDFWriter writer)
   at Scryber.PDF.Layout.PDFLayoutBlock.OutputInnerContent(PDFRenderContext context, PDFWriter writer)
   at Scryber.PDF.Layout.PDFLayoutBlock.DoOutputToPDF(PDFRenderContext context, PDFWriter writer)
   at Scryber.PDF.Layout.PDFLayoutRegion.DoOutputToPDF(PDFRenderContext context, PDFWriter writer)
   at Scryber.PDF.Layout.PDFLayoutBlock.OutputInnerContent(PDFRenderContext context, PDFWriter writer)
   at Scryber.PDF.Layout.PDFLayoutBlock.DoOutputToPDF(PDFRenderContext context, PDFWriter writer)
   at Scryber.PDF.Layout.PDFLayoutRegion.DoOutputToPDF(PDFRenderContext context, PDFWriter writer)
   at Scryber.PDF.Layout.PDFLayoutBlock.OutputInnerContent(PDFRenderContext context, PDFWriter writer)
   at Scryber.PDF.Layout.PDFLayoutBlock.DoOutputToPDF(PDFRenderContext context, PDFWriter writer)
   at Scryber.PDF.Layout.PDFLayoutPage.OutputContent(PDFRenderContext context, PDFWriter writer)
   at Scryber.PDF.Layout.PDFLayoutPage.DoWritePage(PDFRenderContext context, PDFWriter writer, PDFObjectRef parent)
   at Scryber.PDF.Layout.PDFLayoutPage.DoOutputToPDF(PDFRenderContext context, PDFWriter writer)
   at Scryber.PDF.Layout.PDFLayoutDocument.OutputAllPages(PDFObjectRef parent, PDFRenderContext context, PDFWriter writer)
   at Scryber.PDF.Layout.PDFLayoutDocument.OutputPageTree(PDFRenderContext context, PDFWriter writer)
   at Scryber.PDF.Layout.PDFLayoutDocument.WriteCatalogEntries(PDFRenderContext context, PDFWriter writer)
   at Scryber.PDF.Layout.PDFLayoutDocument.WriteCatalog(PDFRenderContext context, PDFWriter writer)
   at Scryber.PDF.Layout.PDFLayoutDocument.DoOutputToPDF(PDFRenderContext context, PDFWriter writer)
   at Scryber.Components.Document.DoOutputToPDF(PDFLayoutDocument layout, PDFRenderContext context, PDFWriter writer)
   at Scryber.Components.Document.DoOutputAndAppendToPDF(PDFLayoutDocument doc, PDFRenderContext context, PDFWriter writer)
   at Scryber.Components.Document.RenderToPDF(PDFRenderContext context, PDFWriter writer)
   at Scryber.Components.Document.RenderToPDF(Stream toStream)
   at Scryber.Components.Document.SaveAsPDF(String path, FileMode mode, Boolean bind)
   at Application.Core.Services.PdfService.SaveInvoiceAsPdf(Invoice invoice) in D:\Dev\project\src\Application.Core\Services\PdfService.cs:line 134
```

**To Reproduce**
- Install Scryber.Core.Mvc 6.0.0-beta8 on a project which includes SixLabors.ImageSharp.Web 2.0.2.
- Add an `<img />` with either a relative or absolute URL in the src-attribute to the HTML template and render the PDF.

I have attached a basic test project to demonstrate this issue.

[ScryberTest.zip](https://github.com/richard-scryber/scryber.core/files/10112634/ScryberTest.zip)

This issue does not occur when using Scryber.Core.Mvc 5.x.

**Comment by creativesuspects** [comment]
Hi Richard, I can confirm that this issues is resolved in v6.0.013-beta! 👍

**Comment by richard-scryber** [comment]
That is awesome, thanks Arjan!
